### DISCUSSION TITLE
Installing Visual C++ 2005 Redistributable

### DISCUSSION THREAD

**Discussion by DavidSanan** [Whisky-App/Whisky#457] [Q&A] [answered]
Hi,

I am trying to install The Secret Of Monkey Island SE from Steam. When I launch it, I get a black screen. According to crossover tips, it is necessary to install Microsoft Visual C++ 2005 (8.0) Redistributable (64-bit) [https://www.codeweavers.com/compatibility/crossover/tips/the-secret-of-monkey-island-special-edition/game-dependencies](url)

This is not supported by MS anymore, and the only thing that I have been able to find is a SP [https://www.microsoft.com/en-in/download/details.aspx?id=26347](url), but after trying to install it, it doesn't work (I am not sure if I need admin mode or similar, and in that case, how to do it in Whisky).

Any idea/tips on how to solve this?

Thanks

**Comment by mavethee** [comment, marked as answer]
Try this package:

https://www.techpowerup.com/download/visual-c-redistributable-runtime-package-all-in-one/

Download, unzip, run .bat in your Whisky bottle.
